examples/browser/package.json

```diff
@@ -24,19 +24,23 @@
     "clean": "rimraf lib",
     "prepare": "npm run clean && npm run build",
     "build": "npm run build:localdeps && npm run build:backend && npm run build:frontend",
+    "build:dev": "npm run build:localdeps:dev && npm run build:backend && npm run build:frontend",
```

Comment:
yes I hesitatated about that... but yes for the examples I think it's better like you say